app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt

```diff
@@ -60,6 +63,13 @@ private fun ScreenContent(
         mutableStateOf(false)
     }
     val context = LocalContext.current
+    val customIntent = CustomTabsIntent.Builder()
+        .setDefaultColorSchemeParams(
+            CustomTabColorSchemeParams.Builder()
```

Comment:
don't set a specific toolbar's color. It should be color by default.

Reply:
why should it be the default color and not changed to the primary color like in the whole app?

Reply:
The entire application must maintain the same style. Application's toolbar has color:  `TopAppBarDefaults.topAppBarColors(containerColor = MaterialTheme.colorScheme.surface)`